diff --git a/prefabs/honey.lua b/prefabs/honey.lua
index 83fe195..30bf349 100644
--- a/prefabs/honey.lua
+++ b/prefabs/honey.lua
@@ -42,10 +42,10 @@ local function fn()
 
     inst:AddComponent("inspectable")
 
-    inst:AddComponent("perishable")
-    inst.components.perishable:SetPerishTime(TUNING.PERISH_SUPERSLOW)
-    inst.components.perishable:StartPerishing()
-    inst.components.perishable.onperishreplacement = "spoiled_food"
+    -- inst:AddComponent("perishable")
+    -- inst.components.perishable:SetPerishTime(TUNING.PERISH_SUPERSLOW)
+    -- inst.components.perishable:StartPerishing()
+    -- inst.components.perishable.onperishreplacement = "spoiled_food"
 
     inst:AddComponent("inventoryitem")
 
diff --git a/prefabs/seeds.lua b/prefabs/seeds.lua
index bac494d..114cb01 100644
--- a/prefabs/seeds.lua
+++ b/prefabs/seeds.lua
@@ -116,10 +116,10 @@ local function common(anim, cookable, oceanfishing_lure)
     MakeSmallPropagator(inst)
     MakeHauntableLaunchAndPerish(inst)
 
-    inst:AddComponent("perishable")
-    inst.components.perishable:SetPerishTime(TUNING.PERISH_SUPERSLOW)
-    inst.components.perishable:StartPerishing()
-    inst.components.perishable.onperishreplacement = "spoiled_food"
+    -- inst:AddComponent("perishable")
+    -- inst.components.perishable:SetPerishTime(TUNING.PERISH_SUPERSLOW)
+    -- inst.components.perishable:StartPerishing()
+    -- inst.components.perishable.onperishreplacement = "spoiled_food"
 
     inst.scrapbook_removedeps = scrapbook_removedeps
 
diff --git a/prefabs/spidergland.lua b/prefabs/spidergland.lua
index 491ee86..47ed08f 100644
--- a/prefabs/spidergland.lua
+++ b/prefabs/spidergland.lua
@@ -39,7 +39,8 @@ local function fn()
     inst:AddComponent("tradable")
 
     inst:AddComponent("healer")
-    inst.components.healer:SetHealthAmount(TUNING.HEALING_MEDSMALL)
+    -- inst.components.healer:SetHealthAmount(TUNING.HEALING_MEDSMALL)
+    inst.components.healer:SetHealthAmount(TUNING.HEALING_LARGE)
 
     return inst
 end
diff --git a/prefabs/veggies.lua b/prefabs/veggies.lua
index af6cc8e..a0e3c79 100644
--- a/prefabs/veggies.lua
+++ b/prefabs/veggies.lua
@@ -444,10 +444,10 @@ local function MakeVeggie(name, has_seeds)
         inst.components.edible.healthvalue = TUNING.HEALING_TINY / 2
         inst.components.edible.hungervalue = TUNING.CALORIES_TINY
 
-        inst:AddComponent("perishable")
-        inst.components.perishable:SetPerishTime(TUNING.PERISH_SUPERSLOW)
-        inst.components.perishable:StartPerishing()
-        inst.components.perishable.onperishreplacement = "spoiled_food"
+        -- inst:AddComponent("perishable")
+        -- inst.components.perishable:SetPerishTime(TUNING.PERISH_SUPERSLOW)
+        -- inst.components.perishable:StartPerishing()
+        -- inst.components.perishable.onperishreplacement = "spoiled_food"
 
         inst:AddComponent("cookable")
         inst.components.cookable.product = "seeds_cooked"
diff --git a/tuning.lua b/tuning.lua
index c7d7716..ff63747 100644
--- a/tuning.lua
+++ b/tuning.lua
@@ -28,9 +28,9 @@ function Tune(overrides)
     local dusk_time = seg_time * dusk_segs
     local night_time = seg_time * night_segs
 
-    local multiplayer_attack_modifier = 1--0.6--0.75
-    local multiplayer_goldentool_modifier = 1--0.5--0.75
-    local multiplayer_armor_durability_modifier = 0.7
+    local multiplayer_attack_modifier = 10--0.6--0.75
+    local multiplayer_goldentool_modifier = 2--0.5--0.75
+    local multiplayer_armor_durability_modifier = 1.5
     local multiplayer_armor_absorption_modifier = 1--0.75
     local multiplayer_wildlife_respawn_modifier = 1--2
 
@@ -49,6 +49,10 @@ function Tune(overrides)
     local OCEAN_NOISE_BASE_SCALE = 10
     local OCEAN_SPEED_BASE_SCALE = 0.01
 
+    local MY_PERISHTIME_MODIFIER = 4
+    local MY_LIGHTTIME_MODIFIER = 3
+    local MY_FULE_MODIFIER = 3
+
     TUNING =
     {
         MODS_QUERY_TIME = 60, -- NOTES(JBK): We do not need live instant updates per query for this while the mod panels are up.
@@ -1589,11 +1593,16 @@ function Tune(overrides)
 
         WET_FUEL_PENALTY = 0.75,
 
-        TINY_FUEL = seg_time*.25,
-        SMALL_FUEL = seg_time * .5,
-        MED_FUEL = seg_time * 1.5,
-        MED_LARGE_FUEL = seg_time * 3,
-        LARGE_FUEL = seg_time * 6,
+        -- TINY_FUEL = seg_time*.25,
+        -- SMALL_FUEL = seg_time * .5,
+        -- MED_FUEL = seg_time * 1.5,
+        -- MED_LARGE_FUEL = seg_time * 3,
+        -- LARGE_FUEL = seg_time * 6,
+        TINY_FUEL = seg_time*.25 * MY_FULE_MODIFIER,
+        SMALL_FUEL = seg_time * .5 * MY_FULE_MODIFIER,
+        MED_FUEL = seg_time * 1.5 * MY_FULE_MODIFIER,
+        MED_LARGE_FUEL = seg_time * 3 * MY_FULE_MODIFIER,
+        LARGE_FUEL = seg_time * 6 * MY_FULE_MODIFIER,
 
         TINY_BURNTIME = seg_time*.1,
         SMALL_BURNTIME = seg_time*.25,
@@ -1623,61 +1632,97 @@ function Tune(overrides)
         COLDFIREPIT_BONUS_MULT = 2,
 
         PIGTORCH_RAIN_RATE = 2,
-        PIGTORCH_FUEL_MAX = night_time,
+        -- PIGTORCH_FUEL_MAX = night_time,
+        PIGTORCH_FUEL_MAX = night_time*MY_FULE_MODIFIER,
 
-        NIGHTLIGHT_FUEL_MAX = (night_time+dusk_time)*3,
+        -- NIGHTLIGHT_FUEL_MAX = (night_time+dusk_time)*3,
+        NIGHTLIGHT_FUEL_MAX = (night_time+dusk_time)*3*MY_FULE_MODIFIER,
         NIGHTLIGHT_FUEL_START = (night_time+dusk_time),
 
         TORCH_RAIN_RATE = 1.5,
-        TORCH_FUEL = night_time*1.25,
+        -- TORCH_FUEL = night_time*1.25,
+        TORCH_FUEL = night_time*1.25*MY_FULE_MODIFIER,
         TORCH_SHRINE_FUEL_RATE_MULT = .2,
 
-        MINIFAN_FUEL = day_time * 0.3,
+        -- MINIFAN_FUEL = day_time * 0.3,
+        MINIFAN_FUEL = day_time * 0.3*MY_FULE_MODIFIER,
 
-        COMPASS_FUEL = day_time * 4,
+        -- COMPASS_FUEL = day_time * 4,
+        COMPASS_FUEL = day_time * 4*MY_FULE_MODIFIER,
         COMPASS_ATTACK_DECAY_PERCENT = -0.3,
 
-        NIGHTSTICK_FUEL = night_time*6,
+        -- NIGHTSTICK_FUEL = night_time*6,
+        NIGHTSTICK_FUEL = night_time*6*MY_FULE_MODIFIER,
         LIGHTER_RAIN_RATE = 1,
-        LIGHTER_FUEL = total_day_time*1.25,
-
-        MINERHAT_LIGHTTIME = (night_time+dusk_time)*2.6,
-        LANTERN_LIGHTTIME = (night_time+dusk_time)*2.6,
-        SPIDERHAT_PERISHTIME = 4*seg_time,
+        -- LIGHTER_FUEL = total_day_time*1.25,
+        LIGHTER_FUEL = total_day_time*1.25*MY_FULE_MODIFIER,
+
+        -- MINERHAT_LIGHTTIME = (night_time+dusk_time)*2.6,
+        MINERHAT_LIGHTTIME = (night_time+dusk_time)*2.6*MY_LIGHTTIME_MODIFIER,
+        -- LANTERN_LIGHTTIME = (night_time+dusk_time)*2.6,
+        LANTERN_LIGHTTIME = (night_time+dusk_time)*2.6*MY_LIGHTTIME_MODIFIER,
+        -- SPIDERHAT_PERISHTIME = 4*seg_time,
+        SPIDERHAT_PERISHTIME = 4*seg_time*MY_PERISHTIME_MODIFIER, -- 거미모자
         SPIDERHAT_RANGE = 12,
-        ONEMANBAND_PERISHTIME = 6*seg_time,
-        ONEMANBAND_RANGE = 12,
+        -- ONEMANBAND_PERISHTIME = 6*seg_time,
+        ONEMANBAND_PERISHTIME = 6*seg_time*MY_PERISHTIME_MODIFIER, -- 원 맨 밴드
+        -- ONEMANBAND_RANGE = 12,
+        ONEMANBAND_RANGE = 30,
         HEATROCK_NUMUSES = 8,
 
-        GRASS_UMBRELLA_PERISHTIME = 2*total_day_time*perish_warp,
-        UMBRELLA_PERISHTIME = total_day_time*6,
-        EYEBRELLA_PERISHTIME = total_day_time*9,
-
-        STRAWHAT_PERISHTIME = total_day_time*5,
-        EARMUFF_PERISHTIME = total_day_time*5,
-        WINTERHAT_PERISHTIME = total_day_time*10,
-        BEEFALOHAT_PERISHTIME = total_day_time*10,
-
-        TRUNKVEST_PERISHTIME = total_day_time*15,
-        REFLECTIVEVEST_PERISHTIME = total_day_time*8,
-        HAWAIIANSHIRT_PERISHTIME = total_day_time*15,
-        SWEATERVEST_PERISHTIME = total_day_time*10,
-        HUNGERBELT_PERISHTIME = total_day_time*8,
-        BEARGERVEST_PERISHTIME = total_day_time*7,
-        RAINCOAT_PERISHTIME = total_day_time*10,
-
-        WALRUSHAT_PERISHTIME = total_day_time*25,
-        FEATHERHAT_PERISHTIME = total_day_time*8,
-        TOPHAT_PERISHTIME = total_day_time*8,
-
-        ICEHAT_PERISHTIME = total_day_time*4,
-        MOLEHAT_PERISHTIME = total_day_time*1.5,
-        RAINHAT_PERISHTIME = total_day_time*10,
-        CATCOONHAT_PERISHTIME = total_day_time*10,
-        GOGGLES_PERISHTIME = total_day_time*10,
-        NUTRIENTSGOGGLESHAT_PERISHTIME = total_day_time*10,
-
-        WALTERHAT_PERISHTIME = total_day_time*10,
+        -- GRASS_UMBRELLA_PERISHTIME = 2*total_day_time*perish_warp,
+        GRASS_UMBRELLA_PERISHTIME = 2*total_day_time*perish_warp*MY_PERISHTIME_MODIFIER,
+        -- UMBRELLA_PERISHTIME = total_day_time*6,
+        UMBRELLA_PERISHTIME = total_day_time*6*MY_PERISHTIME_MODIFIER,
+        -- EYEBRELLA_PERISHTIME = total_day_time*9,
+        EYEBRELLA_PERISHTIME = total_day_time*9*MY_PERISHTIME_MODIFIER, -- 눈우산
+
+        -- STRAWHAT_PERISHTIME = total_day_time*5,
+        STRAWHAT_PERISHTIME = total_day_time*5*MY_PERISHTIME_MODIFIER, 
+        -- EARMUFF_PERISHTIME = total_day_time*5,
+        EARMUFF_PERISHTIME = total_day_time*5*MY_PERISHTIME_MODIFIER, -- 토끼 귀마개
+        -- WINTERHAT_PERISHTIME = total_day_time*10,
+        WINTERHAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER,
+        -- BEEFALOHAT_PERISHTIME = total_day_time*10,
+        BEEFALOHAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, 
+
+        -- TRUNKVEST_PERISHTIME = total_day_time*15,
+        TRUNKVEST_PERISHTIME = total_day_time*15*MY_PERISHTIME_MODIFIER, -- 방풍 조끼
+        -- REFLECTIVEVEST_PERISHTIME = total_day_time*8,
+        REFLECTIVEVEST_PERISHTIME = total_day_time*8*MY_PERISHTIME_MODIFIER, -- 여름 조끼
+        -- HAWAIIANSHIRT_PERISHTIME = total_day_time*15,
+        HAWAIIANSHIRT_PERISHTIME = total_day_time*15*MY_PERISHTIME_MODIFIER, -- 꽃무늬 셔츠
+        -- SWEATERVEST_PERISHTIME = total_day_time*10,
+        SWEATERVEST_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 스웨터 조끼?
+        -- HUNGERBELT_PERISHTIME = total_day_time*8,
+        HUNGERBELT_PERISHTIME = total_day_time*8*MY_PERISHTIME_MODIFIER, -- 허기의 벨트?
+        -- BEARGERVEST_PERISHTIME = total_day_time*7,
+        BEARGERVEST_PERISHTIME = total_day_time*7*MY_PERISHTIME_MODIFIER, -- 곰소리 조끼?
+        -- RAINCOAT_PERISHTIME = total_day_time*10,
+        RAINCOAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 레인 코트
+
+        -- WALRUSHAT_PERISHTIME = total_day_time*25,
+        WALRUSHAT_PERISHTIME = total_day_time*25*MY_PERISHTIME_MODIFIER, -- 태머섄터
+        -- FEATHERHAT_PERISHTIME = total_day_time*8,
+        FEATHERHAT_PERISHTIME = total_day_time*8*MY_PERISHTIME_MODIFIER, -- 깃털모자
+        -- TOPHAT_PERISHTIME = total_day_time*8,
+        TOPHAT_PERISHTIME = total_day_time*8*MY_PERISHTIME_MODIFIER, -- 신사모
+
+        -- ICEHAT_PERISHTIME = total_day_time*4,
+        ICEHAT_PERISHTIME = total_day_time*4*MY_PERISHTIME_MODIFIER, -- 얼음모자
+        -- MOLEHAT_PERISHTIME = total_day_time*1.5,
+        MOLEHAT_PERISHTIME = total_day_time*1.5*MY_PERISHTIME_MODIFIER, -- 두더지 투시경
+        -- RAINHAT_PERISHTIME = total_day_time*10,
+        RAINHAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 방수모
+        -- CATCOONHAT_PERISHTIME = total_day_time*10,
+        CATCOONHAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 고양이 모자
+        -- GOGGLES_PERISHTIME = total_day_time*10,
+        GOGGLES_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 패션고글
+        -- NUTRIENTSGOGGLESHAT_PERISHTIME = total_day_time*10,
+        NUTRIENTSGOGGLESHAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 고급 원예사의 모자
+
+        -- WALTERHAT_PERISHTIME = total_day_time*10,
+        WALTERHAT_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- 월터 모자
 
         GRASS_REGROW_TIME = total_day_time*3,
         SAPLING_REGROW_TIME = total_day_time*4,
@@ -2253,10 +2298,13 @@ function Tune(overrides)
 
         RESURRECT_HEALTH = 50,
 
-        SEWINGKIT_USES = 5,
-        SEWINGKIT_REPAIR_VALUE = total_day_time*5,
+        -- SEWINGKIT_USES = 5,
+        SEWINGKIT_USES = 10,
+        -- SEWINGKIT_REPAIR_VALUE = total_day_time*5,
+        SEWINGKIT_REPAIR_VALUE = total_day_time*5*MY_PERISHTIME_MODIFIER*2,
 
-        SEWING_TAPE_REPAIR_VALUE = total_day_time*5,
+        -- SEWING_TAPE_REPAIR_VALUE = total_day_time*5,
+        SEWING_TAPE_REPAIR_VALUE = total_day_time*5*MY_PERISHTIME_MODIFIER*2,
 
         RABBIT_CARROT_LOYALTY = seg_time*8,
         RABBIT_POLITENESS_LOYALTY_BONUS = seg_time * 4,
@@ -3505,11 +3553,13 @@ function Tune(overrides)
 		WINTERS_FEAST_OVEN_BASE_COOK_TIME = night_time*.3333,
 
         FIRECRACKERS_STARTLE_RANGE = 10,
-        REDLANTERN_LIGHTTIME = total_day_time * 12,
+        -- REDLANTERN_LIGHTTIME = total_day_time * 12,
+        REDLANTERN_LIGHTTIME = total_day_time * 12*MY_LIGHTTIME_MODIFIER,
         REDLANTERN_RAIN_RATE = .25,
         PERDFAN_USES = 9, --tornado costs 2 charges
         PERDFAN_TORNADO_LIFETIME = 2,
-        DRAGONHAT_PERISHTIME = total_day_time, --only consumes while dancing
+        -- DRAGONHAT_PERISHTIME = total_day_time, --only consumes while dancing
+        DRAGONHAT_PERISHTIME = total_day_time*MY_PERISHTIME_MODIFIER, -- ? --only consumes while dancing
         YOTG_PERD_SPAWNCHANCE = .3,
         MAX_WALKABLE_PLATFORM_RADIUS = 4,
         GOOD_LEAKSPAWN_PLATFORM_RADIUS = 9, -- (MAX_WALKABLE_PLATFORM_RADIUS * 0.75) ^2
@@ -4933,7 +4983,8 @@ function Tune(overrides)
 		MINIBOATLANTERN_SPEED = 0.4,
 		MINIBOATLANTERN_ACCELERATION = 0.3*FRAMES,
 		MINIBOATLANTERN_WANDER_DIST = 6,
-        MINIBOATLANTERN_LIGHTTIME = total_day_time*6,
+        -- MINIBOATLANTERN_LIGHTTIME = total_day_time*6,
+        MINIBOATLANTERN_LIGHTTIME = total_day_time*6*MY_LIGHTTIME_MODIFIER,
         MINIBOATLANTERN_BURNTIME = 1.7,
 
 		YOT_CATCOON_SHRINE_SYMBOLS =
@@ -5232,7 +5283,8 @@ function Tune(overrides)
             BLOCK_TELEPORT_ON_HIT_DURATION_VARIANCE = 2,
         },
 
-        MAST_LAMP_LIGHTTIME = (night_time+dusk_time)*2,
+        -- MAST_LAMP_LIGHTTIME = (night_time+dusk_time)*2,
+        MAST_LAMP_LIGHTTIME = (night_time+dusk_time)*2*MY_LIGHTTIME_MODIFIER,
 
         WATERSTREAK_AOE_DIST = 3,
 
@@ -5275,7 +5327,8 @@ function Tune(overrides)
             NAP_LENGTH = seg_time * 4,
         },
 
-        BATNOSEHAT_PERISHTIME = 0.5*total_day_time*perish_warp,
+        -- BATNOSEHAT_PERISHTIME = 0.5*total_day_time*perish_warp,
+        BATNOSEHAT_PERISHTIME = 0.5*total_day_time*perish_warp*MY_PERISHTIME_MODIFIER, -- 워트 모자인가?
         HUNGERREGEN_TICK_RATE = 5,
         HUNGERREGEN_TICK_VALUE = 5 * (calories_per_day*2.5) / (0.5*total_day_time*perish_warp), -- Ensure that this matches the properties above!
 
@@ -5616,7 +5669,8 @@ function Tune(overrides)
         CARNIVAL_CROWKID_CAMPFIRE_WANDER_DIST = 8,
 		CARNIVAL_CROWKID_CAMPFIRE_SIT_DIST = 4,
 
-        CARNIVAL_VEST_PERISHTIME = total_day_time*5,
+        -- CARNIVAL_VEST_PERISHTIME = total_day_time*5,
+        CARNIVAL_VEST_PERISHTIME = total_day_time*5*MY_PERISHTIME_MODIFIER, -- 까악제 조끼
 
 		CARNIVALDECOR_LAMP_ACTIVATE_TIME = seg_time * 2,
 		CARNIVALDECOR_LAMP_TOKEN_TIME = total_day_time,
@@ -5863,7 +5917,8 @@ function Tune(overrides)
 
         MOONSTORM_SPARK_HEALTH = 100,
 
-        MOONSTORM_GOGGLES_PERISHTIME = total_day_time*1,
+        -- MOONSTORM_GOGGLES_PERISHTIME = total_day_time*1,
+        MOONSTORM_GOGGLES_PERISHTIME = total_day_time*1*MY_PERISHTIME_MODIFIER, -- Astroggles, 우주고글
 
         MOONSTORM_SPEED_MOD = .4,
 
@@ -6068,7 +6123,8 @@ function Tune(overrides)
 		TWIN2_MOUTH_CHARGETIMEOUT	= 0.50,
 		TWIN2_TAUNT_CHANCE			= 0.25,
 
-		EYEMASK_PERISHTIME = total_day_time*10,
+		-- EYEMASK_PERISHTIME = total_day_time*10,
+        EYEMASK_PERISHTIME = total_day_time*10*MY_PERISHTIME_MODIFIER, -- eye mast, 눈알 가면
 
         SHIELDOFTERROR_DAMAGE = wilson_attack*1.5,
         SHIELDOFTERROR_ABSORPTION = .8*multiplayer_armor_absorption_modifier,
@@ -6364,8 +6420,10 @@ function Tune(overrides)
 		LIGHTCRAB_HEALTH = 25 * multiplayer_attack_modifier,
         LIGHTCRAB_PERISH_TIME = total_day_time * 5,
 
-        MONKEY_MEDIUM_HAT_PERISHTIME = total_day_time*6,
-        POLLY_ROGERS_HAT_PERISHTIME = total_day_time*6,
+        -- MONKEY_MEDIUM_HAT_PERISHTIME = total_day_time*6,
+        MONKEY_MEDIUM_HAT_PERISHTIME = total_day_time*6*MY_PERISHTIME_MODIFIER, -- 원숭이 모자?
+        -- POLLY_ROGERS_HAT_PERISHTIME = total_day_time*6,
+        POLLY_ROGERS_HAT_PERISHTIME = total_day_time*6*MY_PERISHTIME_MODIFIER, -- vhffl fhwjdml ahwk
 
         POLLY_ROGERS_WALK_SPEED= 7,
         POLLY_ROGERS_RUN_SPEED= 10,
@@ -6867,7 +6925,8 @@ function Tune(overrides)
             },
         },
 
-        VOIDCLOTH_UMBRELLA_PERISHTIME = total_day_time*15,
+        -- VOIDCLOTH_UMBRELLA_PERISHTIME = total_day_time*15,
+        VOIDCLOTH_UMBRELLA_PERISHTIME = total_day_time*15*MY_PERISHTIME_MODIFIER, -- ?
         VOIDCLOTH_UMBRELLA_DAMAGE = wilson_attack,
 		VOIDCLOTH_UMBRELLA_DOME_RADIUS = 16,
 		VOIDCLOTH_UMBRELLA_DOME_RATE = 1.5,
